src/tape/ensemble.py

```diff
@@ -1142,6 +1160,65 @@ def s2n_inter_quartile_range(flux, err):
                 meta=meta,
             )
 
+        # Output standardization
+        if isinstance(batch, EnsembleSeries):
+            if batch.name == self._id_col:
+                batch = batch.rename("result")
+            batch = EnsembleFrame.from_dask_dataframe(batch.to_frame())
+            if len(on) > 1:
+                batch = batch.reset_index()
+                # Need to overwrite the meta manually as the multiindex will be
+                # interpretted by dask as a single "index" column
+                batch._meta = TapeFrame(columns=on + ["result"])
+
+                if by_band:
+                    batch = batch.categorize(self._band_col).pivot_table(
+                        index=on[0], columns=self._band_col, aggfunc="sum"
+                    )
+
+                    # Need to once again reestablish meta for the pivot
+                    band_labels = batch.columns.values
+                    out_cols = []
+                    for col in ["result"]:
+                        for band in band_labels:
+                            out_cols += [(str(col), str(band))]
+                    batch._meta = TapeFrame(columns=out_cols)
+                    # Flatten the columns to a new column per band
+                    batch.columns = ["_".join(col).strip() for col in batch.columns.values]
```

Comment:
Why do we need this `str.strip()` here?

Reply:
don't think it's a need, but seemed good to not preserve any errant whitespace

Reply:
I don't know what is the right principle here: format new column names in a predictable way (if users access them programmatically) or make columns to have nice names (if users would type them).

We never trim column names in other places, maybe it would be better to have more predictable behavior here?

Reply:
I'm happy to remove the strip call, we can always add it later if there are clear use cases for it

Reply:
removed in latest commit